src/Neo/Wallets/Helper.cs

```diff
@@ -126,35 +128,77 @@ public static long CalculateNetworkFee(this Transaction tx, DataCache snapshot,
                     if (md.ReturnType != ContractParameterType.Boolean)
                         throw new ArgumentException("The verify method doesn't return boolean value.");
                     if (md.Parameters.Length > 0 && invocationScript is null)
-                        throw new ArgumentException("The verify method requires parameters that need to be passed via the witness' invocation script.");
+                    {
+                        var script = new ScriptBuilder();
+                        foreach (var par in md.Parameters)
+                        {
+                            switch (par.Type)
+                            {
+                                case ContractParameterType.Any:
+                                case ContractParameterType.Signature:
+                                case ContractParameterType.String:
+                                case ContractParameterType.ByteArray:
+                                    script.EmitPush(new byte[64]);
+                                    break;
+                                case ContractParameterType.Boolean:
+                                    script.EmitPush(true);
+                                    break;
+                                case ContractParameterType.Integer:
+                                    script.Emit(OpCode.PUSHINT256, new byte[Integer.MaxSize]);
+                                    break;
+                                case ContractParameterType.Hash160:
+                                    script.EmitPush(new byte[UInt160.Length]);
+                                    break;
+                                case ContractParameterType.Hash256:
+                                    script.EmitPush(new byte[UInt256.Length]);
+                                    break;
+                                case ContractParameterType.PublicKey:
+                                    script.EmitPush(new byte[33]);
+                                    break;
+                                case ContractParameterType.Array:
+                                    script.Emit(OpCode.PUSHINT256, new byte[Integer.MaxSize]);
+                                    script.Emit(OpCode.NEWARRAY);
```

Comment:
Differs from NeoGo currently and I'm not sure it's correct, likely should just be `NEWARRAY0`.

Reply:
done